lib/game/game.ex

```diff
@@ -3,15 +3,25 @@ defmodule Reactor.Game do
 
   ##client API
 
-  def start_link do
-    GenServer.start_link(__MODULE__, :ok)
+  def start_link(name) do
+    GenServer.start_link(__MODULE__, :ok, name: name)
   end
 
   ##Server Callbacks
 
   def init(:ok) do
-    state = %{}
+    state = %{
+      users: %{}
+    }
 
     {:ok, state}
   end
+
+  def handle_call({:get_users}, _from, state) do
+    {:reply, {:ok, state.users}, state}
```

Comment:
Nit: I often match the state in the function head rather than use the access syntax (up to you though)